README.md

```diff
@@ -48,25 +48,28 @@ This provider's versions are compatible with the following versions of Cluster A
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-|                                    | v1.22 | v1.23 | v1.24 | v1.25 | v1.26 | v1.27 | v1.28 |
-|------------------------------------| ----- | ----- | ----- | ----- | ----- | ----- | ----- |
-| OpenStack Provider v1alpha5 (v0.6) | +     | ✓     | ✓     | ✓     | +     | +     | +     |
-| OpenStack Provider v1alpha6 (v0.7) | +     | ✓     | ✓     | ✓     | ✓     | ✓     | +     |
-| OpenStack Provider v1alpha7        | +     | +     | +     | +     | ✓     | ✓     | ✓     |
+|                                    | v1.25 | v1.26 | v1.27 | v1.28 |
+|------------------------------------| ----- | ----- | ----- | ----- |
+| OpenStack Provider v1alpha5 (v0.6) | ✓     | +     | +     | +     |
+| OpenStack Provider v1alpha6 (v0.7) | ✓     | ✓     | ✓     | +     |
+| OpenStack Provider v1alpha7        | +     | ✓     | ★     | ★     |
```

Comment:
maybe a little picky, but 1.27 seems not testing 
we only test whether 1.28 be upgraded from 1.27?

Reply:
Yes I was wondering how should I handle this. 1.28 is much more tested but we do create one cluster with 1.27 and then upgrade to 1.28. Maybe I should put ✓ instead?

Reply:
yes, I prefer this way..